tests/e2e/specs/modules/optimize/activation.test.js

```diff
@@ -61,21 +61,26 @@ async function proceedToOptimizeSetup() {
 	] );
 }
 
-async function finishOptimizeSetup() {
-	await Promise.all( [
-		page.waitForNavigation(),
-		expect( page ).toClick(
-			'.googlesitekit-setup-module--optimize button',
-			{ text: /Configure Optimize/i }
-		),
-	] );
-	await expect( page ).toMatchElement(
-		'.googlesitekit-publisher-win__title',
-		{ text: /Congrats on completing the setup for Optimize!/i }
-	);
-}
-
 describe( 'Optimize Activation', () => {
+	async function finishOptimizeSetup() {
```

Comment:
I can't explain why this function needs to be moved here, but it fixes timeout issues. 🤷